### ISSUE TITLE
DataCloneError when loading source

### ISSUE THREAD

**Issue by broodjeaap** [dasmoth/dalliance#138]
I'm trying to load a bam/bam.bai file through the sources parameter.
The files are located in the same directory/location on the server and according to the developer console, are being downloaded.
But after its done, the response is 'null'.
It automatically tries again a couple of times with the same result, giving the "DataCloneError" sometimes.

Are there any requirements for this kind of setup to work?

**Comment by dasmoth** [comment]
Sorry to hear you're having trouble.

From the error message, it seems as though there may be a problem with communication between the main Biodalliance code and a Web Worker that recent versions of Biodalliance use to accelerate BAM (and bigBed/Wig) parsing.  However, beyond that I don't have a lot to go on.

    - What version of Biodalliance are you using?
    - Which web browser are you using, and have you tried a different one?
    - Might it be possible to see the track configuration you're using?

One other thing that might be worth trying is to add:

                 maxWorkers: 0

to your top-level Biodalliance configuration, which will disable the Web Worker system entirely.  Even if this doesn't fix your problem, it's possible that it'll lead to a more informative error message.

**Comment by broodjeaap** [comment]
Thank you for the quick response.

To answer you questions:
I'm using biodalliance 0.13
I've tried multiple webbrowsers (Firefox, Chrome and Opera)
I've also tried different tracks, which work when adding them through the 'browse' funcionality.

Adding the "maxWorkers: 0" will be the next thing I will try, the more informative error message will probably help a lot.

Thank again for the (quick) response.
